### ISSUE TITLE
Error when launching run_tracker: `no file /.../evaluation/list.txt` when selecting VOT18 path

### ISSUE THREAD

**Issue by evalliETH** [alanlukezic/d3s#26]
Hi,

I am sorry if this issue of mine looks naive, I am just getting started with this.

When I type the code for running the tracker: `python run_tracker.py segm default_params --dataset vot18 --sequence \media\evalli\HD_EUGI\Resolve_Videos\Elly_coreography.mp4 --debug 1`, an error occurs, saying:

`FileNotFoundError: [Errno 2] No such file or directory: '/home/evalli/d3s/pytracking/evaluation/list.txt'`

I think this is related to the vot18-path selection in 'local.py', because every time I change that path, the error shows it, adding the 'list.txt' file who looks like missing or non-existing.

Do some of you know how to address this issue? Also, where is the VOT18 dataset located?

Thank you in advance,
Eugenio

**Comment by caiyujue** [comment]
You can directly generate the sequence names contained in the VOT file into a txt file, of course the name is list.txt, or you can refer to the _get_sequence_list function on line 51 of evaluation/votdataset.py, but you may need to modify the names of some sequences

**Comment by alanlukezic** [comment]
The dataset where you want to test your tracker has to be in the correct form. For example, see the vot 2018 dataset and how it is loaded in the pytracking toolkit.

**Comment by evalliETH** [comment]
I am still confused.
I have examined both 'vot18dataset.py' and 'votdataset.py' and understood how the dataset is loaded into the tracker. However I don't understand what it means to "generate the sequence names" into this '.txt' file.
I have never used VOT (or any other dataset in this application for that matter), I am learning from scratch so I am sorry if these questions of mine look silly...

Maybe you could suggest some links for me to learn how to use this. I have looked on the VOT page but the documentation didn't really help me..

Thanks again!

**Comment by caiyujue** [comment]
the “list.txt”‘s  content is as follows，it contains the names of 60 sequences of VOT2018，like this
ants1
ants3
bag
ball1
ball2
basketball
birds1
blanket
bmx
bolt1
bolt2
book
butterfly
car1
conduction1
crabs1
crossing
dinosaur
drone1
drone_across
drone_flip
fernando
fish1
fish2
fish3
flamingo1
frisbee
girl
glove
godfather
graduate
gymnastics1
gymnastics2
gymnastics3
hand
handball1
handball2
helicopter
iceskater1
iceskater2
leaves
matrix
motocross1
motocross2
nature
pedestrian1
rabbit
racing
road
shaking
sheep
singer2
singer3
soccer1
soccer2
soldier
tiger
traffic
wiper
zebrafish1

**Comment by evalliETH** [comment]
Yeah, I figured the list should look like this. I have manually created the list, downloaded the annotations from the VOT2018 and copied the containing folders into '/evaluation'. I don't understand if I also have to save, say the 'ants1' sequence (frames) into the '/evaluation' folder or not, and how to do so in a way that the tracker reads them.
I have analyzed the 'data.py' and 'tracker.py' to see how it reads the sequence and it appears as it reads frames. My question is still where are those frames read from.

**Comment by evalliETH** [comment]
I think I have found my error: I saved the frame folder with the wrong name under the sequence folder: it should be named '/color' and it was named automatically when downloading.

**Comment by mrlihellohorld** [comment]
Why is "pytracking/evaluation/local.py" not in the downloaded file?thanks

**Comment by alanlukezic** [comment]
The file should be generated manually by running the following commands:
python -c "from pytracking.evaluation.environment import create_default_local_file; create_default_local_file()"
python -c "from ltr.admin.environment import create_default_local_file; create_default_local_file()"
please read the installation instructions and see the install.sh script for more details.
